frontend/src/types/media.ts

```diff
@@ -122,3 +122,10 @@ export const isMediaDetail = <T extends SupportedMediaType>(
 ): media is DetailFromMediaType<T> => {
   return !!media && media.frontendMediaType === mediaType
 }
+
+export type Metadata = {
+  label: string
+  url?: string
+  value: string
+  isSource?: boolean
```

Comment:
This field seems unused. I couldn't find any other occurrence of `isSource` in the PR.

Reply:
It's used in the follow-up PR, #2851, to determine what analytics event to send.